src/matrix/cblas-wrappers.h

```diff
@@ -26,6 +26,7 @@
 #include "matrix/kaldi-vector.h"
 #include "matrix/kaldi-matrix.h"
 #include "matrix/matrix-functions.h"
+#include "matrix/kaldi-blas.h"
```

Comment:
I just want to note that you need to be careful to make sure that none of the files #include'd by matrix-lib.h #include cblas-wrappers.h recursively, since it exposes the bad symbols via kaldi-blas.h. I don't think that any of them do from having spent some time just now grepping through the source code.

It would in general be preferable to #include kaldi-blas.h only in the .cc files, but we would have to factor out cblas-wrappers.h into a separate cblas-wrappers.cc implementation file, so I think this is good enough.

Reply:
Compiled well with Intel MKL, matrix test passed. If you think that warning is ok, I would remove casting to int in static_assert.